osrparse/replay.py

```diff
@@ -1,79 +1,24 @@
 import lzma
 import struct
 from datetime import datetime, timezone, timedelta
-from typing import List
-from io import TextIOWrapper
+from typing import List, Optional
+import base64
 
 from osrparse.utils import (Mod, GameMode, ReplayEvent, ReplayEventOsu,
-    ReplayEventCatch, ReplayEventMania, ReplayEventTaiko)
+    ReplayEventCatch, ReplayEventMania, ReplayEventTaiko, Key, KeyMania,
+    KeyTaiko)
 from osrparse.dump import dump_replay
 
-class Replay:
-    # first version with rng seed value added as the last frame in the lzma data
-    LAST_FRAME_SEED_VERSION = 20130319
-    _BYTE = 1
-    _SHORT = 2
-    _INT = 4
-    _LONG = 8
-
-    def __init__(self, replay_data: List[ReplayEvent], pure_lzma: bool, decompressed_lzma: bool):
+class _Unpacker:
```

Comment:
it would also be nice to contain the dumping functions in a class like this one, with a name like `_Packer`